src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    //final String Splitter_Symbol = ",";
     public void getStatistic(String fromFileName, String toFileName) {
+        String[][] result = {{"supply", ""},{"buy", ""},{"result", ""}};
```

Comment:
storing numbers as strings is a bad idea, better to calculate them and store them in some variable